analytics/index.yaml

```diff
@@ -101,47 +103,46 @@ paths:
           example: http://example.com/path/to/previous/page.html
         - name: _cvar
           in: query
-          #TODO: I don't understand this parameter or it's format. HALP!
-          #TODO: Confirm serialization format (it's supposed to be Array JSON, but example suggests Array-like Object JSON).
           #TODO: Confirm description of item format.
-          #TODO: Add limits of item values (e.g. max length).
-          #TODO: Link to a page describing custom variables.
+          #TODO: Link to a definition of scopes.
           description: |
             <a href="https://help.piwik.pro/analytics/custom-variables/">Custom variables</a> list set on the ``visit``
             scope.
 
+            Custom variable param is a JSON object where each key is an ID of a custom variable and each key contains
```

Comment:
I think we can skip name of parameter here. This is continuation of description and we already mentioned name of the parametr in first sentence. Value definition is also separated from definition of the object. Maybe we can combine everything this way: 
```
Format: Object serialized with JSON:

  - key - (`string`) Custom variable ID
  - value - Array with:
    - (`string`) Custom variable name, max length: 200 characters
    - (`string`) Custom variable value, max length: 200 characters
```

`cvar` description should be also updated to keep them unified.